test/integration/import_test.py

```diff
@@ -12,6 +12,14 @@ def connection(dsn, user, password, schema):
         yield con
 
 
+@pytest.fixture
+def connection_without_resolving_hostnames(dsn, user, password, schema):
+    with pyexasol.connect(
+        dsn=dsn, user=user, password=password, schema=schema, compression=True, resolve_hostnames=False
+    ) as con:
+        yield con
+
+
```

Comment:
If you want to reuse this fixture in multiple files (inside folder `test/integration`), then you can consider moving this fixture into a file called `test/integration/conftest.py`. This will make the fixtures available to all pytest files in the same folder and below.

Reply:
I know. But I wanted to avoid making bigger changes. Code duplication is acceptable for me.